client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -376,10 +377,24 @@ function getFallbackDestination( {
 			return `/plans/my-plan/${ siteSlug }?thank-you=true&install=all`;
 		}
 
+		const getSimpleThankYouUrl = (): string => {
+			debug( 'attempt to add extra information as url query string' );
+			const titanProducts = cart?.products?.filter( ( product ) => isTitanMail( product ) );
+
+			if ( titanProducts && titanProducts.length > 0 ) {
+				const emails = titanProducts[ 0 ].extra?.email_users;
+
+				if ( emails && emails.length > 0 ) {
+					return `/checkout/thank-you/${ siteSlug }/${ pendingOrReceiptId }?email=${ emails[ 0 ].email }`;
```

Comment:
Is it possible to add a test to cover this case? Maybe somewhere around [here](https://github.com/Automattic/wp-calypso/blob/28bf4a84b956f9ad0df74bc826239173eec31ddc/client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js#L1050)?

Reply:
I would love some quick eyes on the test I just added :)

Reply:
Sorry for the long delay but this looks great. Thank you!